### PR TITLE
Adds three new types of dildo cannon (Standard,Fabricator,Admin) and makes it so small dildos can be put inside of you and the cannon without a bunch more coding.

### PR DESCRIPTION
**Pull request by Mlgdatboiwew**
🆑 Kinky Update
add: Dildo cannons
tweak: Dildo Code
/🆑

Have you ever though "Whats one way I can make dindu mad?" I too had that same idea and thought hes against dildos so why not make it even worse?This patch not only adds a cannon that shoots dildos and works as a weapon but you also can put dildos in your butt!(For non erp intentions).

### CODE REVIEW

**Comment by Mlgdatboiwew** [comment]
-1 spriting is horrible you piece of shit instead of drawing go fuck yourself

**Comment by McBawbaggings** [comment]
no

**Comment by Mlgdatboiwew** [comment]
yeah fuck this guy

File: hippiestation/code/game/objects/items/dildo.dm [hunk 1]
```diff
@@ -21,20 +22,23 @@
...
 	throwforce = 7
+	w_class = 1
 
 /obj/item/dragon/equine
 	name = "Equine Dildo"
 	desc = "Yes, it's the whole horse."
 	icon_state = "equine"
 	force = 11
 	throwforce = 9
+	w_class = 4
```
Comment (ghost) [inline hippiestation/code/game/objects/items/dildo.dm]: use the defines

Comment (Mlgdatboiwew) [reply hippiestation/code/game/objects/items/dildo.dm]: you mean the weight_class_small thing?

**Comment by Pyko1** [comment]
welp not modular enough

**Comment by Mlgdatboiwew** [comment]
I honestly dont know whats with you fuckers and modules make like a tutorial or som shit

File: hippiestation/code/game/objects/items/dildo.dm [hunk 2]
```diff
@@ -1,48 +1,13 @@
-//Coded by Nexendia~ :)
+//Fully modular
 
 /obj/item/dragon
-	name = "Bad Dragon"
```
Comment (ghost) [inline hippiestation/code/game/objects/items/dildo.dm]: why

Comment (Mlgdatboiwew) [reply hippiestation/code/game/objects/items/dildo.dm]: I made it more modular

Comment (ghost) [reply hippiestation/code/game/objects/items/dildo.dm]: you're a fucking idiot

**Comment by MoreRobustThanYou** [comment]
build this

**Comment by tortellinitony** [comment]
No!

**Comment by DinduStuffin** [comment]
-1
I don't think we need a dildo cannon. Pie cannons were good enough. Just use a pneumatic cannon.

**Comment by Mlgdatboiwew** [comment]
Pneumatic cannons are horrible

**Comment by Mlgdatboiwew** [comment]
You also cant put dildos in them

**Comment by banthebantz** [comment]
dildo esword

**Pull request closed without merging**